taxi/forms.py

```diff
@@ -0,0 +1,45 @@
+from django.contrib.auth.forms import UserCreationForm
+from django import forms
+from django.core import validators
+from taxi.models import Driver
+
+
+class DriverCreationForm(UserCreationForm):
+    license_number = forms.CharField(
+        min_length=8,
+        max_length=8,
+        validators=[
+            validators.RegexValidator(
+                regex=r"^[A-Z]{3}\d{5}$",
+                message="The license number must consist"
+                        " of 3 uppercase letters followed by 5 digits.",
+            )
+        ],
+    )
+
+    class Meta:
+        model = Driver
```

Comment:
According to the checklist, you should use `get_user_model()` instead of `Driver` directly. This is a best practice in Django to avoid potential issues with custom user models.